src/test/java/org/folio/eusage/reports/MainVerticleTest.java

```diff
@@ -266,11 +274,43 @@ static void getErmResource(RoutingContext ctx) {
       ar.add(new JsonObject()
           .put("id", kbTitleId)
           .put("name", "fake kb title instance name")
+          .put("identifiers", new JsonArray()
+              .add(new JsonObject()
+                  .put("identifier", new JsonObject()
+                      .put("value", term)
+                  )
+              ))
```

Comment:
A second identifier with some dummy value should be added to ensure that the first identifier is taken.

Reply:
I don't even know which one should be taken.

Reply:
No longer used.